app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,44 @@
+ 
+class AdminsController < ApplicationController
+    #prepend_before_action :require_no_authentication, only: [:cancel]
+    before_action :authorize_admin
+    # skip_before_action :verify_signed_out_user 
+    before_action :remove_password_params_if_blank, only: [:update] 
```

Comment:
Indentation still not fixed.  This can be done automatically using rubocop.  You do `rubocop -a <filename>` for each of the files you want to fix.

Reply:
I ran rubocop and fixed the indentation.